cli_reference/openshift_cli/managing-cli-profiles.adoc

```diff
@@ -0,0 +1,346 @@
+[id="managing-cli-profiles"]
+= Managing CLI profiles
+include::modules/common-attributes.adoc[]
+:context: managing-cli-profiles
+
+toc::[]
+
+== Overview
+A CLI configuration file allows you to configure different profiles, or
+contexts, for use with the xref:../../cli_reference/index.adoc#cli-tools-overview[CLI tools overview]. A context
+consists of xref:../../authentication/understanding-authentication.adoc#understanding-authentication[user authentication]
+and {product-title} server information associated with a _nickname_.
+
+[id="switching-between-cli-profiles"]
+== Switching between CLI profiles
+
+Contexts allow you to easily switch between multiple users across multiple
+{product-title} servers, or _clusters_, when using issuing CLI operations. Nicknames
+make managing CLI configuration easier by providing short-hand references to
+contexts, user credentials, and cluster details.
+
+After xref:../../cli_reference/index.adoc#cli-tools-overview[logging in with the CLI] for the first time,
+{product-title} creates a *_~/.kube/config_* file if one does not already exist. As more authentication and connection details are provided to the
+CLI, either automatically during an `oc login` operation or by xref:manually-configuring-cli-profiles[setting them explicitly], the updated
+information is stored in the configuration file:
+
+.CLI config file
+====
```

Comment:
You can remove the surrounding ==== before and after the code block

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

It looks like a lot of the formatting in this PR was brought over from 3.11, but our style guidelines changed for content going into 4.x